src/renderer/screens/Editor/ImportExportDialog/LoadDefaultKeymap.js

```diff
@@ -0,0 +1,69 @@
+// -*- mode: js-jsx -*-
+/* Chrysalis -- Kaleidoscope Command Center
+ * Copyright (C) 2018, 2019  Keyboardio, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { useState } from "react";
+import path from "path";
+const fs = require("fs");
+
+import Focus from "@chrysalis-api/focus";
+
+import Button from "@material-ui/core/Button";
+
+import { getStaticPath } from "../../../config";
+
+export default function LoadDefaultKeymap({ loadDefault }) {
+  let focus = new Focus();
+  const [device] = useState(focus.device);
+
+  const { vendor, product } = device.info;
+  const cVendor = vendor.replace("/", "");
+  const cProduct = product.replace("/", "");
+  const layoutPath = layout =>
+    path.join(getStaticPath(), cVendor, cProduct, `${layout}.json`);
+
+  const defaultLayouts = ["Qwerty", "Dvorak", "Colemak"];
+  const deviceLayouts = [];
+
+  defaultLayouts.map(layout => {
+    const path = layoutPath(layout);
+    try {
+      fs.accessSync(path);
+      deviceLayouts.push({ name: layout, path });
+    } catch (err) {
+      console.log(`${vendor} ${layout} does not exist`);
+    }
+  });
+
+  return (
+    <div style={{ display: "flex" }}>
+      {deviceLayouts.length > 0 && (
```

Comment:
Would it be possible to return an empty component if `defaultLayouts` is empty? Returning null before this main return seems cleaner than the condition here, and would result in an empty div less for boards that have no default layouts.

Reply:
I believe the original reason I did this was that we may have to adjust the layout in the Dialog component if I returned null in case of empty. This is because we are currently using Flexbox `space-between` to evenly distribute this component apart from the clipboard links and put each to the left/right of this row. I could be completely wrong since I'm talking about CSS off the top of my head. 😂

Reply:
Oh, right, fair. Lets stick to div + condition then, to err on the safe side.

Reply:
Just played with it, this would cause some additional complexity to make the layout work better as we'd have to check for the existence in the parent component and not in `LoadDefaultKeymap` and have a conditional that would modify the CSS there to take care of having a consistent layout.